src/Microsoft.Net.Http.Headers/ContentDispositionHeaderValue.cs

```diff
@@ -64,13 +64,13 @@ public IList<NameValueHeaderValue> Parameters
 
         public string Name
         {
-            get { return GetName(NameString); }
+            get { return Unquote(GetName(NameString)); }
```

Comment:
The final representation is supposed to be quoted. Removing them is just for convenience. It could be cached, but I don't think that's an immediate concern.